go/consensus/tendermint/apps/staking/state/state_test.go

```diff
@@ -27,6 +32,85 @@ func mustInitQuantityP(t *testing.T, i int64) *quantity.Quantity {
 	return &q
 }
 
+func TestDelegationQueries(t *testing.T) {
+	db := dbm.NewMemDB()
+	tree := iavl.NewMutableTree(db, 128)
+	s := NewMutableState(tree)
+
+	fac := memorySigner.NewFactory()
+
+	// Generate escrow account.
+	escrowSigner, err := fac.Generate(signature.SignerEntity, rand.Reader)
+	require.NoError(t, err, "generating escrow signer")
```

Comment:
Can save yourself a lot of typing by doing:
```
  require := require.New(t)

  require.NoError(err, "look ma! I saved having to specify `t` everywhere")
```

Reply:
fixed (throughout whole file)